arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -41,7 +39,7 @@
         "tags": [
           "Namespaces"
         ],
-        "operationId": "Namespaces_ListBySubscription",
+        "operationId": "Namespaces_List",
```

Comment:
@v-Ajnava  It's great to see that you are fixing swagger issues to be more compliant. But as you that these changes include breaking changes for released SDKs we'd highly recommend to do all these improvements when you have new api-version for service bus so that you don't break existing SDK customers unless you have reasons to do it now.